### DISCUSSION TITLE
Particle identifiers

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#12624] [Q&A] [answered]
Dear developers, 

If I understand correctly, Smokeview uses unique particle IDs to track their trajectories. I wonder if these IDs can somehow change during a simulation, e.g. via the ember generation tag **EMBER_PARTICLE=T**, or if other particles leave the simulation domain. 

Thanks in advance!

**Comment by drjfloyd** [comment, marked as answer]
Particle tags do not change. Each particle is assigned a tag that is unique for the entire simulation that starts with the first particle on each mesh being given the mesh number and each subsequent on mesh a number incremented by the total meshes.  If you had five meshes the first particle on each mesh would be numbered:
1 2 3 4 5
the second
6 7 8 9 10
and so on.

**Reply by r-broek** [reply]
Thank you for the explanation!
